src/clang_parser.cpp

```diff
@@ -865,15 +869,15 @@ std::unordered_set<std::string> ClangParser::get_unknown_typedefs()
   return unknown_typedefs;
 }
 
-void ClangParser::resolve_unknown_typedefs_from_btf(BPFtrace &bpftrace)
+void ClangParser::resolve_unknown_types_from_btf(BPFtrace &bpftrace)
 {
   bool check_unknown_types = true;
   while (check_unknown_types)
   {
     // Collect unknown typedefs and retrieve their definitions from BTF.
     // These must be resolved completely since any unknown typedef will cause
```

Comment:
```suggestion
    // Collect unknown types and retrieve their definitions from BTF.
    // These must be resolved completely since any unknown type will cause
```